src/main/scala/nodes/stats/Sampling.scala

```diff
@@ -2,7 +2,7 @@ package nodes.stats
 
 import breeze.linalg.{DenseVector, DenseMatrix}
 import org.apache.spark.rdd.RDD
-import pipelines.{FunctionNode, Transformer}
+import pipelines.FunctionNode
```

Comment:
FWIW We should create a new issue to track this -- our issue tracker could use some love :)